NBitcoin/StandardScriptTemplate.cs

```diff
@@ -148,12 +148,55 @@ protected override bool CheckScriptSigCore(Script scriptSig, Op[] scriptSigOps,
 			throw new NotSupportedException();
 		}
 
+		[Obsolete("Use GenerateWitScript instead")]
 		public Script GenerateScriptSig(TaprootSignature signature)
 		{
 			if (signature == null)
 				throw new ArgumentNullException(nameof(signature));
 			return new Script(Op.GetPushOp(signature.ToBytes()));
 		}
+
+		public PayToTaprootScriptSigParameters ExtractWitScriptParameters(WitScript witScript)
+		{

```

Comment:
need to check witScript isn't null

Reply:
Fixed in https://github.com/MetacoSA/NBitcoin/pull/1082/commits/a24e96ca435db10e225e8eee9e2584ca7fb9ece9.